### PR TITLE
feat: console descriptions

### PR DESCRIPTION
**Pull request by eliobischof**
- Adds text to the console that describes the most important resources purposes and helps deciding on organization and instance settings.
- Adds links to concepts for further reading.
- Cleaned up unused translations as of this PR
```bash
# Translated Keys
for file in src/assets/i18n/*.json; do
    jq -r 'paths(scalars | true) as $p  | [ ( [ $p[] | tostring ] | join(".") )] | join(": ")' $file | sort | uniq > ${file}.translated-keys.tmp
done

# Potentially Unused Keys With The Current Branches Changes 
git diff main -- ./src/app | grep -E '^-.*$' | grep -oh -E '([[:upper:]]|_)+(\.([[:upper:]]|[[:digit:]]|_)+)+' | sort | uniq > /tmp/potentially-used-keys.txt

# Used Keys In Source Files
grep -ohr --exclude-dir=proto -E '([[:upper:]]|_)+(\.([[:upper:]]|[[:digit:]]|_)+)+' src/app/ | sort | uniq > /tmp/used-keys.txt

# Cleanup Translations
for file in src/assets/i18n/*.json; do
    cat ${file}.translated-keys.tmp | jq "del($(comm -23 /tmp/potentially-used-keys.txt /tmp/used-keys.txt | sed 's/^/./g' | tr '
' ',' | sed s/.$//))" && rm $file
done
```

### Deviations from the acceptance criteria

- [ ] For a better user experience, we don't use info boxes. Instead, we use more subtle texts. Info boxes are distracting and annoying for users who have read the texts already.
- [ ] Managers now have at least an Add Manager tooltip. It's probably a bit too subtle, but we couldn't find a better solution.
- [ ] The onboarding view "More Shortcuts" are described, besides that I think more text on the onboarding view is not welcoming.
@hifabienne @adlerhurst FYI

### Tests

I verified manually, that the keys under DESCRIPTION are the same in each translation file.

```bash
# Compare bg.json keys to the en.json keys
# No output means no difference
jq -r '.DESCRIPTIONS | paths(scalars | true) as $p  | [ ( [ $p[] | tostring ] | join(".") )] | join(": ")' bg.json | sort | uniq | comm -3 /tmp/en.json -
```

### Definition of Ready

- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [x] My code has no repetitions
- [x] Critical parts are tested automatically
- [x] Where possible E2E tests are implemented
- [x] Documentation/examples are up-to-date
- [x] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: console/src/assets/i18n/it.json [hunk 1]
```diff
@@ -1,5 +1,269 @@
...
+        "METADATA": "Aggiungi attributi personalizzati al tuo utente come il tuo dipartimento. Puoi utilizzare queste informazioni nelle azioni della tua organizzazione."
+      }
+    },
+    "AUTHORIZATIONS": {
+      "TITLE": "Autorizzazioni",
+      "DESCRIPTION": "Le autorizzazioni definiscono i diritti di accesso di un utente a un progetto. Puoi concedere a un utente l'accesso a un progetto e definire i ruoli degli utenti all'interno di quel progetto."
+    },
+    "ACTIONS": {
+      "TITLE": "Azioni",
+      "DESCRIPTION": "Esegui codice personalizzato sugli eventi che si verificano quando i tuoi utenti si autenticano su ZITADEL. Automatizza i tuoi processi, arricchisci i metadati dei tuoi utenti e i loro token o notifica sistemi esterni.",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:70]:
```suggestion
      "DESCRIPTION": "Esegui script sugli eventi che si verificano quando i tuoi utenti si autenticano su ZITADEL. Automatizza i tuoi processi, arricchisci i metadati dei tuoi utenti e i loro token o notifica sistemi esterni.",
```

File: console/src/assets/i18n/it.json [hunk 2]
```diff
@@ -1,5 +1,269 @@
...
+      "IAM_EVENTS": {
+        "TITLE": "Eventi",
+        "DESCRIPTION": "Questa pagina mostra tutti i cambiamenti di stato nella tua istanza fino al limite dell'audit trail delle tue istanze. Filtra l'elenco per intervallo di tempo per scopi di debug o filtralo per un aggregato per scopi di audit."
+      },
+      "IAM_FAILED_EVENTS": {
+        "TITLE": "Eventi Falliti",
+        "DESCRIPTION": "Questa pagina mostra tutti gli eventi falliti nella tua istanza. Se ZITADEL non si comporta come ti aspetti, controlla sempre prima questa lista."
+      },
+      "IAM_VIEWS": {
+        "TITLE": "Viste",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:122]:
```suggestion
        "TITLE": "Views",
```

File: console/src/assets/i18n/it.json [hunk 3]
```diff
@@ -1,5 +1,269 @@
...
+        "DESCRIPTION": "Questa pagina mostra tutte le tue viste del database e quando hanno elaborato il loro ultimo evento. Se ti mancano alcuni dati, controlla se la vista è aggiornata."
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:123]:
```suggestion
        "DESCRIPTION": "Questa pagina mostra tutte le tue views del database e quando hanno elaborato il loro ultimo evento. Se ti mancano alcuni dati, controlla se la view è aggiornata."
```

File: console/package.json [hunk 4]
```diff
@@ -84,6 +84,7 @@
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-jasmine": "^5.1.0",
     "karma-jasmine-html-reporter": "^2.1.0",
+    "node-jq": "^4.3.1",
```
[the remark this answers is not in the document]
Comment (eliobischof) [reply console/package.json]: its deleted

**Review by peintnermax — APPROVED** [review]

**Pull request merged**